src/components/DataCollector/validateUserEventOptions.js

```diff
@@ -27,6 +27,7 @@ export default ({ options }) => {
     data: objectOf({}),
     documentUnloading: boolean(),
     renderDecisions: boolean(),
+    decisionContext: objectOf({}),
```

Comment:
Since decisionContext is just a basic `objectOf({})` I'm not sure it will add much value to extract it now.  But I agree, definitely should if it becomes more defined/complex.